acquia/acquia_cms
# ACMS-1470: Include Acquia DAM Module

**Motivation**
Fixes #ACMS-1470

**Proposed changes**
Include Acquia DAM Module.

**Alternatives considered**
NA

**Testing steps**
- Install site with this branch
- Enable acquia_cms_dam module
- Verify working fine(out-of-box)

**Merge requirements**
